crepr/crepr.py

```
@@ -335,12 +358,22 @@ def add(
     kwarg_splat: Annotated[str, splat_option] = "{}",
     diff: Annotated[Optional[bool], diff_inline_option] = None,  # noqa: UP007
+    ignore_existing: Annotated[bool | None, ignore_existing_option] = None,
 ) -> None:
     """Add __repr__ to all classes in the source code."""
     for module, file_path in get_modules(files):
-        changes = create_repr(module, kwarg_splat)
+        changes, existing_repr_found = create_repr(module, kwarg_splat)
         if not changes:
             continue
+        if existing_repr_found and not ignore_existing:
+<<<<<<< Updated upstream
```

Ruff

SyntaxError: Expected an indented block after `if` statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Unexpected indentation

---

Line too long (115 > 88)

(E501)